lib/slim_fast/parser.ex

```diff
@@ -95,6 +71,40 @@ defmodule SlimFast.Parser do
     end
   end
 
+  @attr_regex ~r/([\w-]+)\s*=\s*(".+"|[\w-]+)/
```

Comment:
For starters `@user.name`, `user.name` and `user["name"]` support will be great.
Ideally, any code without spaces, leaving `attr=(code with spaces)` case for the future.